channeldb/graph.go

```diff
@@ -1997,12 +1997,54 @@ func delChannelEdge(edges, edgeIndex, chanIndex, zombieIndex,
 		return nil
 	}
 
+	nodeKey1, nodeKey2 := makeZombiePubkeys(&edgeInfo, edge1, edge2)
```

Comment:
do we also want to gate this conditionally on `StrictZombiePruning`? i.e. we will save both pubkeys in non-strict mode to keep the current resurrection behavior? should just be able to keep the old logic in tact.

as is, a non-strict zombie will only be resurrected under the criteria of a strict zombie, regardless of which mode it is run in.

Reply:
Yeah makes sense, ended up threading it through on the router side.